### ISSUE TITLE
Sponge compatibility

### ISSUE THREAD

**Issue by aharabedian** [mcMMO-Dev/mcMMO#3823]
Is there any plan on creating a sponge version of mcMMO? I know that previously (2016) there was no plan. With the mcMMO revamp, is there a possibility?

**Comment by nossr50** [comment]
funny you ask :) 

https://docs.google.com/document/d/1DENmwZrgHfFB7ktSIHs9u2rbmPRXeqHavQZFuzwJilg/edit?usp=sharing

**Comment by aarroonn789** [comment]
Fantastic news! My players are loving the revamp so far and I am looking forward to being able to migrate from Spigot to Sponge.

Keep up the great work!

**Comment by Matbart1** [comment]
> funny you ask :)
> 
> https://docs.google.com/document/d/1DENmwZrgHfFB7ktSIHs9u2rbmPRXeqHavQZFuzwJilg/edit?usp=sharing

I'd like the necro this and see if there has been any development for it in the last 5 years.
I've got a spongeforge server for Tekkit 2 and would LOVE to use MCMMO in it. I read this post and made the assumption that it would have been completed and documented by now for Sponge but I cannot find anything and already purchased the rights to download MCMMO.

**Comment by Chew** [comment]
Yeah, normally I'd say to open a new issue but it's the same feature request.

I don't think there's been any progress on this, but I could be wrong.

**Comment by Matbart1** [comment]
That's too bad! I'm only familiar with full stack development commercially and a little bit of game dev but there is an outdated sponge plug in AdventureMMO that I can look into updating to the new sponge API to make it compatible. Lord knows how long that might take me though.

**Comment by nossr50** [comment]
I'd love to add support for Sponge, but it's just a matter of prioritization and other priorities are winning out. It would be no small task to make mcMMO compatible with Sponge keep in mind.
